### PR TITLE
Adding metadata file used for certification.

### PR DESCRIPTION
**Pull request by jmichelp**

### CODE REVIEW

File: metadata/metadata.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+   "keyProtection" : 1,
+   "attestationRootCertificates" : [
+   ],
+   "aaguid" : "664d9f67-84a2-412a-9ff7-b4f7d8ee6d05",
+   "publicKeyAlgAndEncoding" : 260,
+   "protocolFamily" : "fido2",
+   "upv" : [
+      {
+         "major" : 1,
+         "minor" : 0
```
Comment (kaczmarczyck) [inline metadata/metadata.json:12]: If this is what I think it is, we report different numbers in HID:
`https://github.com/google/OpenSK/blob/3867497c4bab9c2772ba3a59fc4f1675fd83ce0e/src/ctap/hid/mod.rs#L122`

We report serial number `v0.1` over HID to the test tool for some reason. Maybe that's somewhere in the driver?

File: metadata/metadata.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+   "description" : "OpenSK authenticator",
+   "authenticatorVersion" : 1,
+   "isSecondFactorOnly" : false,
+   "userVerificationDetails" : [
+      [
+         {
+            "userVerification" : 1
+         },
+         {
+         	"userVerification" : 4
```
Comment (kaczmarczyck) [inline metadata/metadata.json:37]: nit: indentation differs

File: metadata/metadata.json (hunk 1 above)
Comment (jmichelp) [reply metadata/metadata.json:12]: It's the version of the supported upv protocol.
1.2 means UAF, 1.0 means U2F, etc.

See this [link](https://fidoalliance.org/specs/fido-v2.0-rd-20180702/fido-metadata-statement-v2.0-rd-20180702.html#dictionary-metadatastatement-members).

The shown [FIDO2 Example](https://fidoalliance.org/specs/fido-v2.0-rd-20180702/fido-metadata-statement-v2.0-rd-20180702.html#fido2-example) there also reports 1.0

File: metadata/metadata.json (hunk 2 above)
Comment (jmichelp) [reply metadata/metadata.json]: Nice catch. Tab vs. spaces. That's the drawback of having edited this file with different editors :)

File: metadata/metadata.json (hunk 1 above)
Comment (kaczmarczyck) [reply metadata/metadata.json]: Oh okay, then my comment is unrelated. We might still want to change the version reported in HID. Not in this PR, but in general.

**Review by kaczmarczyck — APPROVED** [review]

**Pull request merged**